### PR TITLE
Run tests/linter on pushes too

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by gijsk** [comment]
@djmitche looks like this is failing with `fatal: repository '{{event.head.repo.url}}' does not exist`. Unsure what's causing that... 😬

**Comment by djmitche** [comment]
Ugh, more things to fix in the quickstart :)

**Comment by djmitche** [comment]
oops, bad indentatation

**Comment by gijsk** [comment]
lol, apparently we're too slow on the VM/hardware on the 2nd wikipedia page test?

**Comment by gijsk** [comment]
I pushed https://github.com/mozilla/readability/commit/b962d0cf8e6a685f6eaffcb4e3e4df466e606421 to (hopefully) deal with the timeout. Gonna try and merge this, but take the task timeout down to 10 minutes as it's currently running in about 3 minutes...

File: .taskcluster.yml [hunk 1]
```diff
@@ -2,25 +2,36 @@ version: 1
...
+  in:
+    $match:
+      '(tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"]) || (tasks_for == "github-push")':
+        taskId:
+          $eval: as_slugid("pr_task")
+        provisionerId: proj-misc
+        workerType: ci
+        deadline: {$fromNow: '1 day'}
+        payload:
+          maxRunTime: 3600
```
Comment (gijsk) [inline .taskcluster.yml:23]:
```suggestion
          maxRunTime: 600
```

**Pull request merged**